front/components/sections/Calculator.tsx

```diff
@@ -0,0 +1,174 @@
+import Button from 'components/items/Button'
+import Paragraph from 'components/items/Paragraph'
+import WrapperWidth from 'components/wrappers/Wrapperwidth'
+import { useFormik } from 'formik'
+import { strapiTherapistGet } from 'lib/strapi/therapists/get'
+import { strapiTherapistQuery } from 'lib/strapi/therapists/queryType'
+import React, {useEffect, useState} from 'react'
+
+const Calculator = () => {
+    const [data, setData] = useState<formData>();
+    const [isSubmit, setIsSubmit] = useState(false);
+    const [cost, setCost] = useState(0);
+    const [discount, setDiscount] = useState(0);
+    const formik = useFormik({
+        initialValues: {
+            therapist: '',
+            session: undefined,
+            relative: '',
+            relativesCode: undefined,
+            workshop: false,
+        },
+        onSubmit: () => {
+            setData(formik.values);
+            setIsSubmit(true);
+        },
+    });
+    const [therapistList, setTherapistList] = useState<strapiTherapistQuery[]>();
+    useEffect(() => {
+        strapiTherapistGet({}).then((res) => {
+            setTherapistList(res.data.data);
+        });
+    }, []);
+    const therapistNameList = therapistList?.map(item => item.attributes.first_name);
+    useEffect(() => {
```

Comment:
In my mind whole costa and discount settings could be done outside of Calculator file. You can create for it it's own function and place it outside of Calculator function, or even better you can remove it from the file and create structure like /sections/calculator/helper.ts, /sections/calculator/index.tsx